debian/patches/drop-unsupported-systemd-condition-environment.patch

```diff
@@ -62,16 +52,16 @@ This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
  ConditionPathExists=!/etc/cloud/cloud-init.disabled
  ConditionKernelCommandLine=!cloud-init=disabled
 -ConditionEnvironment=!KERNEL_CMDLINE=cloud-init=disabled
- 
+
  [Service]
  Type=oneshot
---- a/systemd/cloud-init-hotplugd.socket
-+++ b/systemd/cloud-init-hotplugd.socket
-@@ -8,7 +8,6 @@ Description=cloud-init hotplug hook sock
- After=cloud-config.target
+--- a/systemd/cloud-init-main.service.tmpl
++++ b/systemd/cloud-init-main.service.tmpl
```

Comment:
I don't think that we want to be modifying this service file on focal. I plan on submitting a follow-up PR which will patch it out completely.

Reply:
Updated